splink/comparison_template_library.py

```diff
@@ -1036,7 +1036,7 @@ def __init__(
         self,
         col_name: str,
         invalid_postcodes_as_null=False,
-        valid_postcode_regex="^[A-Z]{1,2}[0-9][A-Z0-9]? [0-9][A-Z]{2}$",
+        valid_postcode_regex="^[A-Za-z]{1,2}[0-9][A-Za-z0-9]? [0-9][A-Za-z]{2}$",
```

Comment:
Should we add an argument that fixes the casing for your column?

Reply:
Hmm, potentially

I have just realised that the current changes won't fix the exact match level

Reply:
Which would get fixed if we cast the column

Reply:
Is this snippet of code used?
```
postcode_col = InputColumn(col_name, sql_dialect=self._sql_dialect)
postcode_col_l, postcode_col_r = postcode_col.names_l_r()
```

Reply:
No, but I have removed in the string comparators pr I'm about to put the pngs in so was going to leave it in this one